admin_guide/cluster_management.rst

```diff
@@ -34,6 +34,11 @@ In addition to the above, if you want to move existing shards to the newly added
 
 	select rebalance_table_shards('github_events');
 
+.. note::
+
+  Citus' shard rebalancing is fully "online," meaning applications communicating with the database do not experience downtime. Read queries to the cluster can continue without any interruption during the process.
```

Comment:
"Fully Online" contradicts "writes are queued up" in my opinion.

This will not be the case in future versions, but currently the rebalancer is not fully online (in my humble opinion).

cc @sumedhpathak

Reply:
Makes sense. Fully online for reading, but delayed for writing. I guess it will *accept* write queries, so in that way it's not like the system is down, but it's not "fully" up either.

Reply:
So, my understanding may be wrong, but writes are *not* queued up for the majority of the time. Inserts can succeed, and applications can read their writes as well. Once we're close to caught up, then writes are blocked, and the cutover happens. @onderkalaci correct me if I'm wrong.

If there isn't a replica identity, then updates/deletes just error out, but inserts still go through.

Reply:
@sumedhpathak Just for full context, my comment was in regards to the old rebalancer. I believe we can claim fully online for the new one, with the slight caveat of a very short lock at the end (similar to how CREATE INDEX CONCURRENTLY operates as compared to regular CREATE INDEX)